kaltura-ui/src/popup-widget/popup-widget.component.scss

```diff
@@ -8,6 +8,10 @@
     border-radius: 4px;
     overflow: hidden;
     box-shadow: 0 16px 20px 0 rgba(0,0,0,0.3);
+    &.transparent {
+        box-shadow: none;
+        background: 0;
```

Comment:
@diamond-darrell - is this compatible with IE11? If not - you can use background-color: rgba(0,0,0,0). If verified to work across all browsers - we can keep it.